### PR TITLE
feat(kv): add dashboards to kv

### LINKED ISSUE #11721: kv: port dashboards
views will also port views

### PR DESCRIPTION
**Pull request by desa**
Closes #11721 

_Briefly describe your proposed changes:_
This PR migrates the bolt dashboard service to the kv store.

  - [ ] Rebased/mergeable
  - [ ] Tests pass
  - [ ] http/swagger.yml updated (if modified Go structs or API)
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: bolt/dashboard.go [hunk 1]
```diff
@@ -12,9 +12,9 @@ import (
 )
 
 var (
-	dashboardBucket         = []byte("dashboardsv2")
-	orgDashboardIndex       = []byte("orgsdashboardsv1")
-	dashboardCellViewBucket = []byte("dashboardcellviewsv1")
+	dashboardBucket         = []byte("dashboards/v2")
```
Comment (goller) [inline bolt/dashboard.go]: This would invalidate all existing dashboards.  I'm ok with keeping the existing names here.

File: kv/dashboard.go [hunk 2]
```diff
@@ -0,0 +1,960 @@
...
+		}
+
+		ds = append(ds, d)
+	}
+
+	return ds, nil
+}
+
+func decodeOrgDashboardIndexKey(indexKey []byte) (orgID influxdb.ID, dashID influxdb.ID, err error) {
+	if len(indexKey) != 2*influxdb.IDLength {
```
Comment (goller) [inline kv/dashboard.go]: I wonder if all these errors in decoding should be internal errors.

File: bolt/dashboard.go (hunk 1 above)
Comment (desa) [reply bolt/dashboard.go]: @goller k

File: kv/dashboard.go (hunk 2 above)
Comment (desa) [reply kv/dashboard.go]: They definitely should be

File: bolt/dashboard.go (hunk 1 above)
Comment (desa) [reply bolt/dashboard.go]: should I also modify the kv buckets to reflect these?

**Comment by goller** [comment]
Yuppers I think they should change too

**Pull request merged**